### PR TITLE
Pull Latest Changes From talon-one:talon_go

### PR DESCRIPTION
**Pull request by SilverCory**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: gomod
+    directory: "/"
+    schedule:
+      interval: weekly
+    reviewers:
+      - "talon-one/sdks-maintainers"
```
Comment (lewcinho) [inline .github/dependabot.yml:8]: if we're gonna keep the dependabot, we need to change the above.

File: go.mod [hunk 2]
```diff
@@ -1,5 +1,5 @@
-module github.com/NandosUK/talon_go/v7
+module github.com/talon-one/talon_go/v8
 
 go 1.13
 
-require golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
+require golang.org/x/oauth2 v0.23.0
```
Comment (SilverCory) [inline go.mod:5]: Conflict, used talon-one/talon_go version

**Review by lewcinho — COMMENTED** [review]
i can't see the `blockReason` anywhere in the code (it's in the docs). has it definitely been added?

File: go.mod (hunk 2 above)
Comment (lewcinho) [reply go.mod:5]: dw too much about listing all the dependency conflicts

**Pull request closed without merging**